src/plugins/template/tests/functional/op_reference/nms_rotated.cpp

```diff
@@ -0,0 +1,497 @@
+        result << "bType=" << param.boxes.type;
+        result << "_bShape=" << param.boxes.shape;
+        result << "_sType=" << param.scores.type;
+        result << "_sShape=" << param.scores.shape;
+        result << "_esiType=" << param.expectedSelectedIndices.type;
+        result << "_esiShape=" << param.expectedSelectedIndices.shape;
+        result << "_escType=" << param.expectedSelectedScores.type;
+        result << "_escShape=" << param.expectedSelectedScores.shape;
+        result << "_evoType=" << param.expectedValidOutputs.type;
+        result << "_evoShape=" << param.expectedValidOutputs.shape;
+        if (param.testcaseName != "") {
+            result << "_=" << param.testcaseName;
+        }
+        return result.str();
+    }
+
+private:
+    static std::shared_ptr<Model> CreateFunction(const NMSRotatedParams& params) {
+        const auto boxes = std::make_shared<opset1::Parameter>(params.boxes.type, params.boxes.shape);
+        const auto scores = std::make_shared<opset1::Parameter>(params.scores.type, params.scores.shape);
+        const auto max_output_boxes_per_class =
+            std::make_shared<opset1::Constant>(params.maxOutputBoxesPerClass.type,
+                                               params.maxOutputBoxesPerClass.shape,
+                                               params.maxOutputBoxesPerClass.data.data());
+        const auto iou_threshold = std::make_shared<opset1::Constant>(params.iouThreshold.type,
+                                                                      params.iouThreshold.shape,
+                                                                      params.iouThreshold.data.data());
+        const auto score_threshold = std::make_shared<opset1::Constant>(params.scoreThreshold.type,
+                                                                        params.scoreThreshold.shape,
+                                                                        params.scoreThreshold.data.data());
+        const auto nms = std::make_shared<opset13::NMSRotated>(boxes,
+                                                               scores,
+                                                               max_output_boxes_per_class,
+                                                               iou_threshold,
+                                                               score_threshold,
+                                                               false,
+                                                               params.expectedSelectedIndices.type,
+                                                               params.clockwise);
+        const auto f = std::make_shared<Model>(nms->outputs(), ParameterVector{boxes, scores});
+        return f;
+    }
+};
+
+class ReferenceNMSRotatedTestWithoutConstants : public ReferenceNMSRotatedTest {
+public:
+    void SetUp() override {
+        auto params = GetParam();
+        function = CreateFunction(params);
```

Comment:
```suggestion
        model = CreateModel(params);
```

Reply:
The "function" is a member of the `CommonReferenceTest` class:

https://github.com/openvinotoolkit/openvino/blob/5611f7d65c4ee0b6a78ed460d0ab093c7f559321/src/plugins/template/tests/functional/op_reference/base_reference_test.hpp#L35

Reply:
I mixed up the classes
https://github.com/openvinotoolkit/openvino/pull/19366/files#diff-811d4ce9d2f38671aadc9df3cac63c93f37b3a8d54c9a089446a90f154040293R34